src/main/java/edu/ucsb/cs156/example/controllers/UCSBDiningCommonsMenuItemController.java

```diff
@@ -0,0 +1,109 @@
+package edu.ucsb.cs156.example.controllers;
+
+import edu.ucsb.cs156.example.entities.UCSBDiningCommons;
+import edu.ucsb.cs156.example.entities.UCSBDiningCommonsMenuItem;
+import edu.ucsb.cs156.example.errors.EntityNotFoundException;
+import edu.ucsb.cs156.example.repositories.UCSBDiningCommonsMenuItemRepository;
+import edu.ucsb.cs156.example.repositories.UCSBDiningCommonsRepository;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.extern.slf4j.Slf4j;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@Tag(name = "UCSBDiningCommonsMenuItem")
```

Comment:
In the future, I will suggest using plurals for API calls (and everywhere else except entities) to retain consistency. In this case, "UCSBDiningCommonsMenuItems", "UCSBDiningCommonsMenuItemsController", etc.

It's an annoying convention, so I'll let it pass this time. Since you have maintained the singular noun everywhere it should still work as intended.

Reply:
Originally, I used plurals for all my files. Later, I noticed that the issue in kanban board required singular usage. Therefore, I updated all my file names and created a new branch for a cleaner commit tree.

`(3) In a separate window, create a new file called UCSBDiningCommonsMenuItemController.java in the src/main/java/edu/ucsb/cs156/example/controllers directory.`

Reply:
Understandable. I was being nitpicky anyway; I also didn't like how the documentation was inconsistent. Good work on this one.